.github/workflows/_build.yml

```diff
@@ -185,15 +179,16 @@ jobs:
         id: screenshotTests
         uses: reactivecircus/android-emulator-runner@v2
         with:
-          target: google_apis
           profile: Nexus 4
           sdcard-path-or-size: 512M
           api-level: ${{ env.api-level }}
           force-avd-creation: false
           emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
           disable-animations: true
           script: |
-            sudo pip3 install Pillow==9.0.1
+            touch logcat.log
```

Comment:
we didn't have logs for failures which made it difficult to debug issues. this adds the logs